eosio.system/src/producer_pay.cpp

```diff
@@ -90,15 +90,11 @@ namespace eosiosystem {
          auto new_tokens = static_cast<int64_t>( (continuous_rate * double(token_supply.amount) * double(usecs_since_last_fill)) / double(useconds_per_year) );
```

Comment:
it looks like this is still calculated as 5% (comments even indicate that).  Which we then divide by 5 to create 1%.  In the future, this will make a brittle constant to adjust.   We should probably make the constant reflect the inflation and leave distribution of that inflation to the code as opposed to the current iteration where you cannot derive actual inflation without mentally executing maths.